src/lua/compat.c

```diff
@@ -5,8 +5,40 @@
  */
 
 #include <lua/compat.h>
+#include "serializer.h"
+
+int
+lbox_msgpuck_escape_forward_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool esc_slash = lua_toboolean(L, -1);
+	lua_pop(L, 1);
+
+	if (esc_slash)
```

Comment:
I just wonder whether there is an option to implement the similar approach you've provided for JSON below? I hardly remember the reasons why do we need to change `char2escape` in place in this case.